zerver/models.py

```diff
@@ -438,6 +438,15 @@ class Realm(models.Model):
         EMAIL_ADDRESS_VISIBILITY_MODERATORS,
     ]
 
+    SHOW_EDITED_AND_MOVED = 1
+    SHOW_MOVED_ONLY = 2
+    SHOW_NO_HISTORY = 3
+    SHOW_MESSAGE_EDIT_HISTORY_TYPES = [
+        SHOW_EDITED_AND_MOVED,
+        SHOW_MOVED_ONLY,
+        SHOW_NO_HISTORY,
+    ]
```

Comment:
We have a convention for these names. This should be defined just before the setting these values are for, and be named e.g. `MESSAGE_EDIT_HISTORY_VISIBILITY_NONE`, `MESSAGE_EDIT_HISTORY_VISIBILITY_MOVES_ONLY`, `MESSAGE_EDIT_HISTORY_VISIBILITY_ALL`.